### ISSUE TITLE
Weird warning

### ISSUE THREAD

**Issue by cristianoccazinsp** [DirtyHairy/async-mutex#14]
I had to update node to 14.x, and after running `npm start -- --reset-cache` I'm getting the following warning: 

```
warn Package async-mutex has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in app/node_modules/async-mutex/package.json
```

Ideas?

**Comment by DirtyHairy** [comment]
Hi @cristianoccazinsp ,

I cannot reproduce your issue by simply importing `async-mutex` in Node 14. Can you explain what exactly `npm start` is doing in your project, or do you maybe have a link to your repository?

Cheers
    -Christian

**Comment by cristianoccazinsp** [comment]
Hello,

It's a react native project. However, I ended up reverting to node 12 since
it was causing various other issues and the warning went away.

**Comment by DirtyHairy** [comment]
Hi @cristianoccazinsp ,

Glad to know that your issue is solved. However, if there is an issue with `async-mutex` in combination with Node 14, I would very much like to resolve it it --- is the affected project publicly available so that I can try to diagnose the issue?

Cheers
    -Christian

**Comment by cristianoccazinsp** [comment]
It's not publicly available, but any react native project with that module
as a requirement should trigger it. Again, only with node 14 for some
reason. Perhaps there was something cached, and everything seemed to work
fine regardless of the warning

**Comment by macfanatic** [comment]
I'm seeing this warning at boot in my react native (expo managed sdk 37) project on node v13.12.0

**Comment by DirtyHairy** [comment]
OK, I just checked and can reproduce the warning by adding `async-mutex` to a react-native project. This seems to be an issue with react-native; a relevant upstream report is https://github.com/facebook/react-native/issues/28710 . I'll release a workaround later this week.

**Comment by DirtyHairy** [comment]
I'm sorry I didn't quite meet my schedule. I've released a fix with 0.2.3

Cheers
    -Christian
